src/nodes_io/reader_tcp.rs

```diff
@@ -45,19 +45,21 @@ impl Parameterizable for TcpReader {
 #[async_trait]
 impl ProcessingNode for TcpReader {
     async fn pull(&self, frame_number: u64, context: &ProcessingContext) -> Result<Payload> {
-        self.notifier.wait(|x| *x >= frame_number).await;
+        self.notifier.wait(move |x| *x >= frame_number).await;
```

Comment:
Hmm we should think about whether we want to make most sources serialized like this or multithreaded like the `WebcamSource`. Multithreaded is probably nicer, but also more complicated. Do we do enough processing after the sources that serialized sources are "good" enough?